src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -151,9 +152,51 @@ impl VirtSandbox {
             resource_configs.push(vm_rootfs);
         }
 
+        // prepare pcie port device configs
+        if let Some(port_configs) = self.prepare_pcie_port_devices().await {
+            resource_configs.extend(port_configs);
```

Comment:
Is there any reason to use `extend()` instead of `push()`?

Reply:
As `port_configs` is a list of port configs whose types are ResourceConfig, and `resource_configs` is also a list `vec![ResourceConfig]`, it will use extend instead of `push`.